pendulum/tz/timezone.py

```diff
@@ -24,10 +24,10 @@ class Timezone(tzinfo):
     POST_TRANSITION = 'post'
     TRANSITION_ERROR = 'error'
 
-    def __init__(self, name, transitions,
-                 tzinfos,
-                 default_tzinfo_index,
-                 utc_transition_times):
+    def __init__(self, name=None, transitions=(),
+                 tzinfos=(),
+                 default_tzinfo_index=0,
+                 utc_transition_times=[]):
```

Comment:
It's dangerous to use mutable default parameters. I'd suggest using `None` values and converting to the proper defaults within `__init__`.